### PR TITLE
feat: v13 `Code Samples`.

### PR DESCRIPTION
**Pull request by WiseDevHelper**
**Please describe the changes this PR makes and why it should be merged:**

_(Split from #708)_

[Demo](https://deploy-preview-721--discordjs-guide.netlify.app/popular-topics/permissions.html)

This PR updates the `Code Samples` of the guide for the `v13` release. It also updates other files to maintain consistency.

**Status**
- [x] Update `Code Samples`.
  - [x] Additional Info.
    - [x] Rest API.
  - [x] Command Handling. 
    - [x] ~~Adding features.~~ (Removed. Refer https://github.com/discordjs/guide/pull/683#issuecomment-866417742)
    - [x] File setup.
  - ~~Commando~~ (To be removed in #711) 
  - [x] Creating your bot.
    - [x] Getting your bot up and running.
    - [x] Configuration files.
    - [x] Adding more commands. 
    - [x] ~~Commands with user input.~~ (Removed. Refer https://github.com/discordjs/guide/pull/683#issuecomment-866417742)
  - [x] Event Handling.
  - [x] Keyv.
  - [x] Miscellaneous.
    - [x] Parsing mention arguments.
  - OAuth. (Does not require updating)
  - [x] Popular topics.
    - [x] Canvas.
    - [x] Permissions. (#705)
    - [x] Reactions. (#729)
    - [x] Webhooks. (#700)
  - [x] Sequelize.
    - [x] Storing data.
    - [x] Currency system. 
  - [x] Sharding. (#695)

Please let me know if there is more to change. I'll be happy to work on them.

### CODE REVIEW

File: code-samples/sequelize/currency/13/app.js [hunk 1]
```diff
@@ -86,17 +85,18 @@ client.on('message', async message => {
...
-				.map((user, position) => `(${position + 1}) ${(client.users.cache.get(user.user_id).tag)}: ${user.balance}💰`)
-				.join('\n'),
-			{ code: true },
+			Formatters.codeBlock(
+				currency.sort((a, b) => b.balance - a.balance)
+					.filter(user => client.users.cache.has(user.user_id))
+					.first(10)
+					.map((user, position) => `(${position + 1}) ${(client.users.cache.get(user.user_id).tag)}: ${user.balance}💰`)
+					.join('\n')
+			)
```
Comment (Danktuary) [inline code-samples/sequelize/currency/13/app.js:97]:
```suggestion
					.join('\n'),
			),
```

File: guide/sequelize/currency.md [hunk 2]
```diff
@@ -350,20 +350,21 @@ For users to search for an item without caring about the letter casing, you can
...
-		.map((user, position) => `(${position + 1}) ${(client.users.cache.get(user.user_id).tag)}: ${user.balance}💰`)
-		.join('\n'),
-	{ code: true },
+	Formatters.codeBlock(
+		currency.sort((a, b) => b.balance - a.balance)
+			.filter(user => client.users.cache.has(user.user_id))
+			.first(10)
+			.map((user, position) => `(${position + 1}) ${(client.users.cache.get(user.user_id).tag)}: ${user.balance}💰`)
+			.join('\n')
+	)
```
Comment (Danktuary) [inline guide/sequelize/currency.md:367]:
```suggestion
			.join('\n'),
	),
```

**Review by Danktuary — COMMENTED** [review]
Looks good! 🎉 

The sharding code samples were updated in #695, and voice doesn't have any code samples files, so only the reactions file is left on your task list

**Comment by WiseDevHelper** [comment]
Moved Reactions to #729 since it should have a PR of it's own.

**Review by Danktuary — APPROVED** [review]

File: code-samples/additional-info/rest-api/13/index.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ client.once('ready', () => {
 	console.log('Ready!');
 });
 
-client.on('message', async message => {
+client.on('messageCreate', async message => {
 	if (!message.content.startsWith(prefix) || message.author.bot) return;
 
 	const args = message.content.slice(prefix.length).trim().split(/ +/);
```
Comment (Danktuary) [inline code-samples/additional-info/rest-api/13/index.js:17]: The `.send(file)` and `.send(embed)` lines in this file need to be updated to their respective `MessageOptions` properties

**Review by Danktuary — APPROVED** [review]

**Pull request merged**